test-common/src/main/kotlin/SakTestData.kt

```diff
@@ -32,3 +36,41 @@ fun Sak.hentGjeldendeVilkårOgGrunnlag(
         },
     )
 }
+
+fun nySakUføre(
+    clock: Clock = fixedClock,
+    søknadsInnhold: SøknadInnhold = søknadinnhold(fnr),
+): Pair<Sak, Søknad.Journalført.MedOppgave> {
+    return nySak(
+        clock = clock,
+        søknadsinnhold = søknadsInnhold,
+    )
+}
+
+fun nySakAlder(
+    clock: Clock = fixedClock,
+    søknadsInnhold: SøknadInnhold = søknadsinnholdAlder(),
+): Pair<Sak, Søknad.Journalført.MedOppgave> {
+    return nySak(
```

Comment:
Hvis ikke man kan spesifisere type, kanskje en assert på disse for å sjekke om det er alder/uføre

Reply:
Endret til spesifisert søknadsinnholdtype